templates/admin.html

```diff
@@ -55,184 +55,10 @@
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
-          </tr>
-          <tr onclick="#">
-            <td>Harry (1B-03)</td>
-            <td>Dispatcher / Officer / Civilian</td>
-            <td>12th February 2018</td>
-            <td>3:02 (HR:MIN)</td>
+            <td>{{user} (1B-03)</td>
```

Comment:
Think you missed something here. I'll end up generating this table myself, so really just leave one column with the variables and I'll handle the rest.